python/ray/train/callbacks/callback.py

```diff
@@ -1,13 +1,17 @@
 import abc
 from typing import List, Dict
 
-from ray.train.callbacks.results_preprocessors import ResultsPreprocessor
+from ray.train.callbacks.results_preprocessors import ResultsPreprocessor, \
+    ExcludedKeysResultsPreprocessor, SequentialResultsPreprocessor
+from ray.train.constants import PROFILER_KEY
 
 
 class TrainingCallback(abc.ABC):
     """Abstract Train callback class."""
 
     results_preprocessor: ResultsPreprocessor = None
+    RESERVED_KEYS = {}
+    ALL_RESERVED_KEYS = {PROFILER_KEY}
```

Comment:
It's not immediately obvious what's the difference between those two variables. Add a comment?

Reply:
Oops yeah haven't documented everything yet. Wanted to make sure overall structure looked good first with this pattern.